java/piranha/src/test/resources/config/properties_test_argument.json

```diff
@@ -0,0 +1,18 @@
+{
+  "piranhaMethodProperties": [
+    {
+      "methodName": "isToggleDisabled",
+      "flagType": "control"
+    },
+    {
+      "methodName": "isFlagTreated",
+      "flagType": "treated",
+      "argumentIndex": 0
```

Comment:
can we add a check where argumentIndex is either 1 or 2?

Reply:
Good catch.